gotocompany/compass/v1beta1/service.proto

```diff
@@ -816,6 +832,52 @@ message SearchAssetsRequest {
   }];
 }
 
+
+message GroupAssetsRequest {
+
+  string groupby = 1 [
+    (validate.rules).string = {
+      ignore_empty: false,
+    },
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "fields used to group assets. multiple fields can also be provided here"
+    }
+  ];
+
+  map<string, string> filter = 2 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+    description: "filter result based on a (nested) field of the asset. the nested field is written with period separated field name. eg, \"filter[data.landscape]\""
+  }];
+
+  string include_fields = 3 [(validate.rules).string = {
+    ignore_empty: true
+  }];
+
+  uint32 size = 4 [
```

Comment:
Along with `size`, should we also support `offset`?

Reply:
We are using this size field in TermsAggregation which takes the size input and not offset. https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html
We can however take the offset here and run multiple queries on es based on offset.

Reply:
Okay, on second thought, `offset` is not as useful here since each 'group' can have an unbounded quantity of assets.